gpkit/tests/t_examples.py

```diff
@@ -82,6 +83,37 @@ def test_simpleflight(self, example):
             sol_rat = sol["sensitivities"]["constants"][key]/consenscheck[key]
             self.assertTrue(abs(1-sol_rat) < 1e-2)
 
+    # TODO: uncomment when breakdown is merged
+    # def test_breakdown_example(self, example):
+    #     if gpkit.units:
+    #         self.assertAlmostEqual(mag(example.sol["cost"]), 8.448, 3)
+    #         self.assertAlmostEqual(mag(example.sol("w2")), 6.448, 3)
+    #     else:
+    #         self.assertAlmostEqual(example.sol["cost"], 5, 4)
+    #         self.assertAlmostEqual(example.sol("w2"), 3, 4)
+
+    def test_LCS_ex(self, example):
+        self.assertAlmostEqual(example.sol["cost"], .5)
```

Comment:
👍 

also `self.assertEqual(len(m.variables_byname("y")), 1)` (assuming `mFull` will get renamed to `m` due to pylint)